server/ucontext-portable.h

```diff
@@ -0,0 +1,37 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#ifndef UCONTEXT_PORTABLE_H
+#define UCONTEXT_PORTABLE_H
```

Comment:
Why not `#pragma once` as usual? I think it would be more clear

Reply:
done